tests/rptest/scale_tests/partition_balancer_scale_test.py

```diff
@@ -182,7 +182,10 @@ def test_node_operations_at_scale(self, type):
             message_size = 256 * (2**10)
             message_cnt = 819200
             consumers = 8
-            partitions_count = 18000
+            # Multiply default partition per shard limit by cores in system, subtract
+            # a few to leave room for the consumer offsets etc partitions.
+            partitions_count = 1000 * self.redpanda.get_node_cpu_count() * len(
+                self.redpanda.nodes) - 32
```

Comment:
Shouldn't we be dividing by the replication factor?

Reply:
Sorry, itchy merge finger -- I think was looking at a github view that had updated test statuses but not shown comments.

You are right!  Followup PR here https://github.com/redpanda-data/redpanda/pull/9274